### ISSUE TITLE
Question: onAccessibilityEscape example?

### ISSUE THREAD

**Issue by markbrown4** [facebook/react-native#24270]
`onAccessibilityEscape` doesn't seem to be firing for the simplest use case on iOS.
I'm using the 2 finger Z shaped scrub and no dice.

Is there a bug with it or am I not using it correctly?

```
class AccessibleApp extends React.Component {
  state = {
    escaped: false,
  };

  render() {
    return (
      <View style={{flex: 1}} accessible onAccessibilityEscape={() => this.setState({escaped: true})}>
        <Text>{`Escaped: ${this.state.escaped}`}</Text>
      </View>
    );
  }
}
```

In particular I'm trying to use it with the React Native's Modal component to dismiss but haven't managed to get it to do anything so far.

**Comment by markbrown4** [comment]
🙄

**Comment by elucaswork** [comment]
@markbrown4 I just tested it on my side, and it works (using your code).
> [will be called when someone performs the "escape" gesture, which is a two finger Z shaped gesture](https://facebook.github.io/react-native/docs/accessibility#onaccessibilityescape-ios)

with this gesture, `onAccessibilityEscape` is called.

**Comment by markbrown4** [comment]
Yep, this was broken in `1.59` and fixed in `1.60`
